src/ServiceControl.Transports.SQS/AmazonSQSQuery.cs

```diff
@@ -292,4 +292,6 @@ public override string SanitizeEndpointName(string endpointName)
             return endpointName;
         }
     }
+
+    public override string SanitizedEndpointNameCleanser(string endpointName) => endpointName.ToLower();
```

Comment:
I checked SQS and the queue name is case sensitive.
```suggestion

```

Reply:
removed it